source/snes9x/port.h

```diff
@@ -216,10 +216,10 @@
 #define RIGHTSHIFT_int8_IS_SAR
 #define RIGHTSHIFT_int16_IS_SAR
 #define RIGHTSHIFT_int32_IS_SAR
-#ifndef GEKKO
+#ifndef __LIBRETRO__
```

Comment:
I'm pretty sure we don't want SNES_JOY_READ_CALLBACKS for Wii and I'm certain we don't want multi-format.